packages/iTwinUI-react/src/core/Table/filters/FilterToggle.test.tsx

```diff
@@ -0,0 +1,68 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { FilterToggle } from './FilterToggle';
+import { tableFilters } from './tableFilters';
+
+it('should display active filter', () => {
+  const column = {
+    canFilter: true,
+    filterValue: 'value',
+    render: jest.fn(),
+    setFilter: jest.fn(),
+    Filter: tableFilters.TextFilter(),
+  } as any;
```

Comment:
Don't use `any`, it is against our linting rules.

Reply:
Also, why you need to cast it at all?

Reply:
If I don't cast it, I'm instructed to add 28 additional props which aren't used in the test.